src/guide/expressions.md

```diff
@@ -81,77 +81,88 @@ below.
-});
-```
+In this case, Handlebars will pass the link helper two parameters: the String `See Website` and the value of
+`people.url` from the below provided input `people` object.
 
-You could use the exact same helper with dynamic text based on the value of `story.text`:
+<ExamplePart examplePage="/examples/helper-multiple-parameters.md" show="input" />
 
-```handlebars
-{{link story.text story.url}}
-```
+The helper function `link` is used to generate a hyperlink as described in the script.
+
+<ExamplePart examplePage="/examples/helper-multiple-parameters.md" show="preparationScript" />
+
+We will obtain the HTML link output using the input parameters
+
+<ExamplePart examplePage="/examples/helper-multiple-parameters.md" show="output" />
+
+In the above example, You could use the exact same helper with dynamic text based on the value of `people.text`:
+
+<Flex>
+<ExamplePart examplePage="/examples/helper-dynamic-parameters.md" show="template" />
+<ExamplePart examplePage="/examples/helper-dynamic-parameters.md" show="input" />
+</Flex>
+
+### `Helpers with Hash arguments`
 
 Handlebars helpers can also receive an optional sequence of key-value pairs as their final parameter (referred to as
 hash arguments in the documentation):
 
-```handlebars
-{{link "See more..." href=story.url class="story"}}
-```
+<ExamplePart examplePage="/examples/helper-hash-arguments.md" show="template" />
+
+In tha template,the final parameter `href=people.url class="people"` are hash arguments sent to the helper.
 
 The keys in hash arguments must each be simple identifiers, and the values are Handlebars expressions. This means that
 values can be simple identifiers, paths, or Strings.
 
-```js
-Handlebars.registerHelper("link", function(text, options) {
-  var attrs = [];
+If we pass the below input to the template, the value of `people.url` can be obtained from the `people` object.
 
-  for (var prop in options.hash) {
-    attrs.push(Handlebars.escapeExpression(prop) + '="' + Handlebars.escapeExpression(options.hash[prop]) + '"');
-  }
+<ExamplePart examplePage="/examples/helper-hash-arguments.md" show="input" />
 
-  return new Handlebars.SafeString("<a " + attrs.join(" ") + ">" + Handlebars.escapeExpression(text) + "</a>");
-});
-```
+As described in the helper script below, the hash arguments can be obtained from the last parameter `options` for
+further processing within the helper.
+
+<ExamplePart examplePage="/examples/helper-hash-arguments.md" show="preparationScript" />
+
+The output of above helper is generated as below
+
+<ExamplePart examplePage="/examples/helper-hash-arguments.md" show="output" />
 
 Handlebars provides additional metadata, such as Hash arguments, to helpers as a final parameter.
 
 Handlebars also offers a mechanism for invoking a helper with a block of the template. Block helpers can then invoke
 that block zero or more times with any context it chooses.
 
-[Learn More: Block Helpers](block_helpers.html)
```

Comment:
If you use `!button[Learn More...](block-helpers.html)` you'll get an orange button that looks like the other `Learn More`-links. Could you do that?